backend/tests/integration/editor/test_editor_views.py

```diff
@@ -951,3 +951,72 @@ def test_refextract_text_dedupe_references(mock_refs, inspire_app):
     assert response.status_code == 200
     assert validate(references, subschema) is None
     assert len(references) == 1
+
+
+def test_authorlist_url(inspire_app):
+    schema = load_schema("hep")
+    subschema = schema["properties"]["authors"]
+    user = create_user(role=Roles.cataloger.value)
+    es_response = {
+        "_shards": {"failed": 0, "skipped": 0, "successful": 5, "total": 5},
+        "hits": {"hits": [], "max_score": None, "total": 0},
+        "timed_out": False,
+        "took": 4,
+    }
+
+    with requests_mock.Mocker() as requests_mocker:
+        requests_mocker.register_uri(
+            "GET",
+            "https://arxiv.org/pdf/1612.06414.pdf",
+            content=pkg_resources.resource_string(
+                __name__, os.path.join("fixtures", "1612.06414.pdf")
+            ),
+        )
+        requests_mocker.register_uri(
+            "GET",
+            "http://test-indexer:9200/records-hep/hep/_search?_source=control_number",
+            json=es_response,
+        )
```

Comment:
can't we do it with vcr instead? I would say it is very `next` way of writing tests :p